test/k8sT/manifests/lrp-test.yaml

```diff
@@ -0,0 +1,136 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: k8s1-backend
+  labels:
+    id: be1
+    role: backend
+spec:
+  containers:
+  - name: web
+    image: docker.io/cilium/echoserver:1.10.1
+    imagePullPolicy: IfNotPresent
+    ports:
+    - containerPort: 80
```

Comment:
This won't work, and we need to check if/how CI test is passing. When we have multiple ports, the ports need to be named, and they need to match the ones specified in the pod spec.  The port names are then used to match frontend ports to the corresponding backend ports. 
This is documented in gsg, CRD spec in addition to the code.
@Weil0ng Were you able to run this test successfully?

Reply:
Fixed, I can run the test steps manually, but my local e2e test setup is broken (not due to this PR)...

Reply:
Let's get a confirmation from the automated CI run.